### PR TITLE
Added profile summary

### PR DESCRIPTION
**Pull request by anju54**
I have added profile summary section. Please review it.

### CODE REVIEW

**Comment by anju54** [comment]
pro-summ1
profile_summ

File: src/modules/resume/profile-summary/variants/basic/render.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      <div style={{
+        ...styles.root,
+        backgroundColor: theme.value.color,
+        color: theme.value.contrast
+      }}>
+        <h1 style={styles.headlineText}>{headlineText.value}</h1>
+        <hr />
+        {
+          paragraphs.value.map((paragraph) => {
+            return <div >
```
Comment (srikanthbhamidipalli) [inline src/modules/resume/profile-summary/variants/basic/render.js:34]: Better to add a key prop @anju54

**Comment by AKAspanion** [comment]
@anju54 How does this component differ from 'Introduction'?

**Comment by anju54** [comment]
> @anju54 How does this component differ from 'Introduction'?

Meaning is similar but better way to tell the resume summary. In many places we can see summary so thought of adding this. If this is not okay then please suggest me another component I will try adding that.

**Comment by ManojSatishkumar** [comment]
This is same as introduction

**Pull request closed without merging**